packages/login_client_flutter/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_secure_storage: ^4.2.0
-  login_client: ^2.0.1
+  login_client: ^2.0.3
```

Comment:
This won't pass tests as this version isn't yet published. Please move it to a separate Pull Request.